### PR TITLE
added select field for food name so that users will not be able to enter a food that doesn't exist in our database

### PR DESCRIPTION
**Pull request by dayhahm**
Now the users can choose a food from a dropdown menu that is searchable via selectize. This avoids any issues where a user enters a food that we don't have and therefore won't be able to calculate the co2 input for. Addresses #28

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -35,20 +35,19 @@ limitations under the License.
...
-      <artifactId>selectize.js</artifactId>
-      <version>0.12.5</version>
-    </dependency>
-
     <dependency>
       <groupId>javax.servlet</groupId>
       <artifactId>javax.servlet-api</artifactId>
       <version>3.1.0</version>
       <scope>provided</scope>
     </dependency>
```
Comment (Ivyzhang1998) [inline pom.xml:44]: Out of curiosity, any reason for choosing jstl instead of selectize?

File: src/main/webapp/WEB-INF/views/eaten-food-form.jsp [hunk 2]
```diff
@@ -15,31 +15,50 @@
...
+         <input type="radio" name="mealType" value="dinner">dinner<br>
+         <input type="radio" name="mealType" value="snack">snack<br><br/>
+         <br/><br/>
+         <label for="amount">Amount eaten:</label>
+         <input type="number" name="amount" id="amount" required>
+         <br/>
+         <button>Submit</button>
+     </form>
+     <script>
+         $('select').selectize({
```
Comment (NickChapman) [inline src/main/webapp/WEB-INF/views/eaten-food-form.jsp:62]: You might want to wrap this in a document onready or something like that.

**Review by NickChapman — COMMENTED** [review]
Just one little comment. Have you tried it out? How does it look?

**Review by Ivyzhang1998 — APPROVED** [review]
looks good!

File: pom.xml (hunk 1 above)
Comment (dayhahm) [reply pom.xml:44]: it does use selectize. I think i added jstl as a dependency because selectize needs it

**Pull request merged**